Cargo.toml

```diff
@@ -30,6 +30,12 @@ toml = "0.7.4"
 once_cell = "1.17.0"
 serde_yaml = "0.9.16"
 build_html = "2.2.0"
+uuid = { version = "1.3.3", features = ["v4"] }
+base64 = "0.21.2"
+fastrand = "1.9.0"
+log = "0.4.18"
+pretty_env_logger = "0.5.0"
+dotenvy = "0.15.7"
```

Comment:
`uuid`: Used for spoofing the device clients
`base64`: Used for generating the HTTP Basic Auth header
`fastrand`: Used to generate random devices to spoof
`log`, `pretty_env_logger`, `dotenvy`: Used to improve logging